### PR TITLE
expressvpn.rb: update to 4.0.4

### PR DESCRIPTION
**Pull request by paulbreslin**

### CODE REVIEW

File: Casks/expressvpn.rb [hunk 1]
```diff
@@ -12,5 +13,5 @@
   uninstall :pkgutil => 'com.expressvpn.ExpressVPN',
             :quit => 'com.expressvpn.ExpressVPN'
 
-  depends_on :macos => '>= :yosemite'
-end
+  depends_on :macos => '>= 10.8'
+end
```
Comment (vitorgalvao) [inline Casks/expressvpn.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

File: Casks/expressvpn.rb [hunk 2]
```diff
@@ -12,5 +13,5 @@
   uninstall :pkgutil => 'com.expressvpn.ExpressVPN',
             :quit => 'com.expressvpn.ExpressVPN'
 
-  depends_on :macos => '>= :yosemite'
-end
+  depends_on :macos => '>= 10.8'
```
Comment (vitorgalvao) [inline Casks/expressvpn.rb]: We prefer to use the release symbol.

File: Casks/expressvpn.rb [hunk 3]
```diff
@@ -1,8 +1,9 @@
 cask :v1 => 'expressvpn' do
-  version '4.0.1.288'
-  sha256 '720f03e3c64e3d84a18f92e5055eadb7df6dd75feee831330be50802a836bc16'
+  version '4.0.4.305'
+  sha256 'acb898d8fc1688cc1bbb6f2f1a76f6b3f985543649c25bf1ae16c3e786bf857c'
 
-  url "https://www.expressvpn.com/4568726248/mac/expressvpn-install_v#{version}.pkg"
+  # cloudfront.net is the official download host per the vendor homepage
+  url "https://d11cv8afjim9ws.cloudfront.net/clients/mac/expressvpn-install_v#{version}.pkg"
```
Comment (vitorgalvao) [inline Casks/expressvpn.rb]: Show me where the url is on the website, please, so I can confirm it.

Comment (fanquake) [inline Casks/expressvpn.rb]: That URL is correct, although as far as I can tell it's not available unless you sign into the site

Comment (vitorgalvao) [inline Casks/expressvpn.rb]: > That URL is correct, although as far as I can tell it's not available unless you sign into the site

That’s what I was thinking. That being the case, [this needs to be removed](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#unofficial-vendorless-or-walled-builds).

**Pull request closed without merging**